### ISSUE TITLE
Warnings in MacOS when go-getting and executing

### ISSUE THREAD

**Issue by jmaeso** [go-gl/gl#101]
Hi, when getting the packages I get a bunch of warnings like this one:
```
ld: warning: text-based stub file /System/Library/Frameworks//OpenGL.framework/OpenGL.tbd and library file /System/Library/Frameworks//OpenGL.framework/OpenGL are out of sync. Falling back to library file for linking.
```
I guess this is something related just to macOS.

Btw: Not sure if related but executing [this](https://play.golang.org/p/vbv0KZmgDKC) simple script returns me: `OpenGL version:  4.1 INTEL-10.34.27` (should be 3.3 right?)

System:
Macbook Pro (2016).
MacOS High Sierra Version 10.13.5
Graphics card: Intel Iris Graphics 540 1536 MB

**Comment by jmaeso** [comment]
I'm using 1.10.3
Could be related to the opengl instalation? :S

**Comment by PeterPetrik** [comment]
could be related XCode update? (http://sd.jtimothyking.com/2018/07/26/stub-file-and-library-file-out-of-sync)

**Comment by dmitshur** [comment]
Yes, this appears to be related to that. Ever since Xcode 10 came out, it has been affecting the `crypto/x509` package too, see https://github.com/golang/go/issues/26073.

**Comment by Noofbiz** [comment]
I'm getting this error as well, but I'm still able to use [engo](https://github.com/EngoEngine/engo) and [ebiten](https://github.com/hajimehoshi/ebiten) rendering just fine. The warnings don't seem to have an effect on gl working for me.

**Comment by pwaller** [comment]
Looks like this was fixed over on golang/go#26073. Please ping the thread or file a new issue if not.
